docs/branching.md

```diff
@@ -35,5 +37,12 @@ This would leave all changed files as "changes to be committed" and same as ```g
 
 
 # Additional
+Below are a bunch of commands that might become handy when you want to keep track of the branches and commits.
+
+```git branch --all``` ```git branch``` is a powerful command that can create, check, or delete branches. Use ```git branch``` with the ```--all``` flag to list all local branches and remote branches you chose to track.
```

Comment:
Inline code block only needs a single ` (instead of 3). Could you please remove the extra ones?